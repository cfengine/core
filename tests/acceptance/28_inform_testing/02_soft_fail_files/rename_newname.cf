body file control
{
  inputs => { "$(sys.policy_entry_dirname)/../common_soft_fail.cf.sub" };
}

bundle common testcase
{
  vars:
    "filename" string => "$(this.promise_filename)";
}

bundle agent setup
{
  files:
    "$(G.testfile)"
      create => "true";
    "$(G.testroot)/TEST.foobar"
      delete => init_delete;
}

bundle agent main
{
  files:
    "$(G.testfile)"
      rename => newname;
}

body rename newname
{
  newname => "TEST.foobar";
}
