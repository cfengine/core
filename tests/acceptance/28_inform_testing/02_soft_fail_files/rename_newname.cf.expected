    info: files promise '/tmp/TEST.cfengine' repaired, rename => newname
    info: Renamed file '/tmp/TEST.cfengine' to 'TEST.foobar'
